test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -552,6 +630,12 @@ void UcxContext::destroy_worker()
     }
 
     ucp_worker_destroy(_worker);
+    if (_use_epoll) {
```

Comment:
if not using epoll , i'd expect _epoll_fd would be -1
in fact, maybe remove _use_epoll field, and always check _epoll_fd>=0 instead?

Reply:
I've removed the ```_use_epoll``` and only use ```_epoll_fd``` here.